src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx

```diff
@@ -0,0 +1,536 @@
+            </td>
+            <td>
+                https://insights-collector.newrelic.com/v1/accounts/ACCOUNT_ID/events (Replace `ACCOUNT_ID` with your account id) 
+            </td>
+            <td>
+                Your [NR insights insert API key](https://one.newrelic.com/admin-portal/api-keys/insightkeys) 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                `nr_insights_query`:
+            </td>
+            <td>
+                Allows you to query New Relic custom events
+            </td>
+            <td>
+                `http`
+            </td>
+            <td>
+                https://insights-api.newrelic.com/v1/accounts/ACCOUNT_ID/query (Replace `ACCOUNT_ID` with your account id) 
+            </td>
+            <td>
+                Your [NR insights query API key](https://one.newrelic.com/admin-portal/api-keys/insightkeys) 
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+Once you've configured the four above, you need to configure the Snowflake connection. Snowflake allows you to query for failed test rows. There are [many ways](https://airflow.apache.org/docs/apache-airflow-providers-snowflake/stable/connections/snowflake.html) to configure a snowflake connection. To configure using a private key pair, fill in the following attributes:
+* `Type`: Snowflake
+* `Login`: Your Snowflake username
+* `Account`: Your Snowflake account 
+* `Warehouse`: Your Snowflake warehouse 
+* `Role`: Your Snowflake role. The role must have access to all the DBs used in dbt Cloud to get all failed test rows. 
+* `Private Key Text`: The full private key used for this connection.
+* `Password`: Pass phrase for the private key if it's encrypted. Blank if it's unencrypted.  
+</Step>
+<Step>
+Complete the setup by enabling the `new_relic_data_pipeline_observability_get_dbt_run_metadata2` DAG.
+</Step>
+</Steps> 
+        </TabsPageItem>
+    </TabsPages>
+</Tabs>
+
+
+
+
+## Find your data [#find-data]
+
+This integration creates and reports three custom events to New Relic:
+
+<CollapserGroup>
+<Collapser
+    id="collapser-1-in-group-2"
+    title="`dbt_job_run`"
+>
+
+`dbt_job_run`: Provides metadata and the status of all completed runs. This event does not include any data about models, snapshots, seeds and tests. Attributes include:
+* `project_name`
+* `environment_name`
+* `run_team`
+* All fields listed in the [dbt Cloud v2 API for runs](https://docs.getdbt.com/dbt-cloud/api-v2#/operations/Retrieve%20Run). 
+
+All attributes other than `project_name` and `environment_name` are prepended with `run_`
+
+Example queries:
+```sql
+-- Get status of all job runs in the past seven days
+select 
+    project_name,
+    environment_name,
+    job_name,
+    run_created_at,
+    run_run_duration_humanized,
+    run_status,
+    run_status_humanized,
+    run_status_message
+from dbt_job_run
+since 7 days ago
+```
+</Collapser>
+
+<Collapser
+  id="collapser-1-in-group-2"
+  title="`dbt_resource_run`"
```

Comment:
The backticks are showing in the preview. Was that intended? They look good inside the collapser.